.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*]
```

Comment:
i don't think that we should have this file in repo

Reply:
EditorConfig is a very common file in repos that lets any editor know the standards that a repository follows. In this case, it says "we use LF as line endings". It is common to enforce LF as line endings so that windows machines do not suffer from configuration issues related to CRLF + Git. This file says "when saving a file (no matter how it was checked out), save the line endings as LF so it is checked in correctly no matter what the user's Git configuration is.

https://editorconfig.org/